src/components/RemovableTag.js

```diff
@@ -1,23 +1,51 @@
-function RemovableTag({ tag, onClick = undefined }) {
+function RemovableTag({ tag, onClick = undefined, feed = false, unremovable = false }) {
```

Comment:
Since this is no longer strictly a removable tag, lets change this component to be names something similar to Tag Container.

Reply:
Renamed the component to TagContainer.